dev-cpp/gtest/gtest-1.14.0.ebuild

```diff
@@ -47,7 +47,14 @@ multilib_src_configure() {
 		-Dgmock_build_tests=$(usex test)
 		-Dgtest_build_tests=$(usex test)
 	)
-	use test && mycmakeargs+=( -DPython3_EXECUTABLE="${PYTHON}" )
+	if use test; then
+		if use arm && [[ $(tc-is-softfloat) =~ (softfp)|(no) ]]; then
+			replace-flags -O* -O1 # bug #925093
```

Comment:
Why only for tests? Are we sure it's a test-only issue in the arm case?

For the x86 one, I can accept it's likely a test-only issue though.

Reply:
OK. Then a revbump would be in order.

Reply:
Yes please.